src/Components/Tittle/Tittle.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const Tittle = ({ value, style }) => (
+  <div className={style}>
```

Comment:
if u create ur own component, please add them possibility to be modified more(i mean paste ...wrapperProps and ...titleProps)